pkg/controller/controller.go

```diff
@@ -182,7 +182,6 @@ func (c *Controller) init() {
 		MaxHTTPPort:        c.cfg.MaxHTTPPort,
 		ACL:                c.cfg.ACLEnabled,
 		DefaultTrafficType: c.cfg.DefaultMode,
-		MaeshNamespace:     c.cfg.Namespace,
```

Comment:
Why this removal?

Reply:
If I'm not wrong this property is not used in the provider.

Reply:
it looks like you are right!